next.config.js

```diff
@@ -47,6 +47,14 @@ module.exports = withSwingset({
   })({
     webpack(config) {
       config.plugins.push(HashiConfigPlugin())
+      // disable `exports` support for framer-motion. This will cause Webpack to
+      // fall back to using `module`
+      config.module.rules.push({
+        test: /framer-motion/,
+        resolve: {
+          exportsFields: [],
+        },
+      })
```

Comment:
🤔 Are there any downsides to disabling this that you know of? Seems like no, but just thinking out loud!